### PR TITLE
[CCTRI-1028] Query for Indicator (convert data from C1fApp into CTIM)

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: tests/unit/api/test_enrich.py [hunk 1]
```diff
@@ -76,24 +92,42 @@ def valid_json_multiple():
...
+        assert response["data"]["sightings"]["docs"][0]["observed_time"]
+        assert response["data"]["sightings"]["docs"][0]["schema_version"]
+        assert response["data"]["sightings"]["docs"][0]["type"]
+
+        assert response["data"]["indicators"]["docs"][0]["confidence"]
+        assert response["data"]["indicators"]["docs"][0].pop("id")
+        assert response["data"]["indicators"]["docs"][0]["tlp"]
+        assert response["data"]["indicators"]["docs"][0]["schema_version"]
+        assert response["data"]["indicators"]["docs"][0]["type"]
+        assert response["data"]["indicators"]["docs"][0]["producer"]
```
Comment (AlinaBlahun) [inline tests/unit/api/test_enrich.py:130]: Do we really need all this asserts? It looks like you have whole response check below, so only ones with pop should stay

Comment (AnastasiiaRozlyvan) [reply tests/unit/api/test_enrich.py]: Yes, it’s better to remove them. Thanks, it's done now!

File: tests/unit/api/test_enrich.py [hunk 2]
```diff
@@ -76,24 +82,32 @@ def valid_json_multiple():
...
+        indicators = response['data']['indicators']
+        assert indicators['count'] == 1
 
-        response = response.get_json()
+        sightings = response['data']['sightings']
+        assert sightings['count'] == 1
 
         assert response['data']['sightings']['docs'][0].pop('id')
 
+        assert response["data"]["indicators"]["docs"][0].pop("id")
```
Comment (AlinaBlahun) [inline tests/unit/api/test_enrich.py:110]: Let's do the same quotes at least in one method.

Comment (AnastasiiaRozlyvan) [reply tests/unit/api/test_enrich.py]: Fixed

**Pull request merged**